lib/AccessBuilder.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace Tokenio;
+
+
+use Io\Token\Proto\Common\Token\TokenRequestPayload;
+
+class AccessBuilder extends TokenRequestBuilder
```

Comment:
I am not sure about the name `AccessBuilder`. In java we call it `AccessBuilder` because it is an inner class of the `TokenRequest` class, here we probably should give it the full name. Same for `TransferBuilder`. @gianluca-token Opinion?